docs-md/operate-and-deploy/installation/check-ksqldb-server-health.md

```diff
@@ -80,4 +80,29 @@ If you're running ksqlDB server in a Docker container, run the
 the `ksql-server` container is `Up`. Check the health of the process in
 the container by running `docker logs <ksql-server-container-id>`.
 
-Page last revised on: {{ git_revision_date }}
+Check a headless ksqlDB Server running in a Docker Container 
```

Comment:
Is this recommendation Docker-specific? Seems like it would be applicable for native deployments as well.

Reply:
@mikebin I reworked this section to be more general.